default.go

```diff
@@ -151,6 +153,20 @@ func init() {
 			}
 			return reflect.ValueOf(&intsRanger{from: from, to: to})
 		})),
+		"dump": reflect.ValueOf(Func(func(a Arguments) (result reflect.Value) {
+			runtime := a.runtime
+			vars := runtime.variables
+			var sb strings.Builder
+			sb.WriteString("--dump start\n")
+			for k, v := range vars {
+				sb.WriteString(k)
+				sb.WriteString("=")
+				fastprinter.PrintValue(&sb, v)
```

Comment:
For a debugging function like dump(), I'd say it's fine to just `fmt.Fprintf("%#v\n", a.runtime.variables)` and be done with it. Simply using the tried-and-tested fmt package means there's no reason to explicitly test the dump() output formatting in jet.